.circleci/config.yml

```diff
@@ -151,10 +151,14 @@ jobs:
       - run:
           name: Install poetry dependencies
           command: |
-            # Remove this pin once the upstream Poetry issue is fixed:
+            sudo pip3 install poetry
+            # This flag is important, due to an open upstream Poetry issue:
             # https://github.com/python-poetry/poetry/issues/7184
-            sudo pip3 install poetry==1.2.2
-            poetry install
+            poetry install --no-ansi
+      - run:
+          name: Install test dependencies
+          command: |
+            sudo apt-get install -y libqt5gui5 --no-install-recommends
```

Comment:
Why is this one needed? Is this for testing the built `.deb` (which will use PySide2)? If so, isn't this already a depednency of the produced `.deb`?

Reply:
There is no built `.deb` in that step. We are in a development environment in this case, so we install dependencies via Poetry. The problem here is that PySide needs to load some `.so` libraries, which must be provided via a system package.

Feel free to check the commit message as well for this addition. If it doesn't adequately explain why, I can extend it, and maybe add a comment there.

Reply:
Sounds good. I should have checked it.